packages/connectivity/src/http-agent/http-agent.ts

```diff
@@ -134,9 +172,15 @@ function getMtlsOptions(destination: Destination): Record<string, any> {
     );
   }
   if (mtlsIsEnabled(destination)) {
+    if (registerDestinationCache.mtls.useMtlsCache) {
+      return registerDestinationCache.mtls.getMtlsOptions();
+    }
+    const getCert = readFile(process.env.CF_INSTANCE_CERT as string, 'utf8');
+    const getKey = readFile(process.env.CF_INSTANCE_KEY as string, 'utf8');
+    const [cert, key] = await Promise.all([getCert, getKey]);
```

Comment:
While it might look cleaner, I think the benefit of `Promise.all` is that it's more efficient than using `await` twice in a row, I might be wrong though.

Reply:
I've looked a little bit more into it, with `Promise.all` we read concurrently, with the await call it would be sequentially, therefore `Promise.all` is better